resources/config.toml

```diff
@@ -14,13 +14,21 @@ default_locale='fr-fr'
   sqlite_path = "resources/test.db"
 
 [server]
-  PORT = 8888  # will be lower cased when reading.
-  base_url = "http://localhost:8888/"
+  [server.dev]
+  PORT = 8887  # will be lower cased when reading.
+  base_url = "http://localhost:8887/"
   upload_dir = "uploads/"
   consent = 'resources/consent.html'
   num_days_for_stale = 1.0
 
 [messaging]
+  [messaging.dev]
```

Comment:
Maybe it would be worth syncing with master as this is still using the dev/prod config version.

Reply:
my bad. i ll fix it